docs/data-access/daco/applying.md

```diff
@@ -0,0 +1,106 @@
+| ----------------------- | --------------------------------------- | --------------------- | ---------------- | ---------------------- | ---------------------- |
+| **Applicant/PI** | Yes, if the same as the application submitter | Yes | Yes | Yes | Yes |
+| **Institutional Representative** | No | Yes | No | Yes | No |
+| **Collaborators** | No  | Yes | Yes | No | Yes |
+| **Application Submitter** | Yes  | Yes, but only if the same as the Applicant/PI | Yes, for login purposes | Yes, but only if the same as the Applicant/PI | Yes, but only if the same as the Applicant/PI |
+
+
+## Application Dashboard and Form Basics
+
+1. To begin the application process, log in to [daco.icgc-argo.org](https://daco.icgc-argo.org/).  
+> For authorization, we require a valid Google email address (Gmail or GSuite). If you are applying as the Applicant (Principal Investigator), this will be the email address you will use to log in to ICGC DACO, ICGC ARGO, and ICGC 25K and will be the email address associated with ICGC Controlled Data access.
+1. Once you log in, you will be taken to your applicant dashboard. This is where you can create a new application and access the applications you have in progress. Click on the **Start a new Application** button and you will be taken to a blank application form. 
+![DACO Empty Dashboard](/assets/data-access/daco-noapplications.png)
+1. Complete the entire application form, including all required fields.
+1. You will notice the application form automatically saves as you fill it in. In the table of contents, icons will indicate which sections haven’t been started (blank), are complete ![DACO Section Complete](/assets/data-access/daco-toc-complete.png) or are incomplete ![DACO Section Errors](/assets/data-access/daco-toc-errors.png).
+1. Above the application form basic information about the application is displayed. This is also where you can access actions, such as downloading a pdf, and a progress bar that indicates what stage your application is in. This progress indicator is also displayed on the dashboard card for each application you have started. [Read more about the progress stages below](/docs/data-access/daco/applying#application-progress-stages).
+1. You must have all required sections completed in order to access the final `Sign & Submit` section. [Read more about this below](/docs/data-access/daco/applying#signing-and-submitting-the-application).
+1. Once your application form is finalized and signed by both the Applicant (Principal Investigator) and Institutional Representative, you can upload the signed application PDF in the `Sign & Submit` section and click the **Submit Application** button. Read more about this below.
+
+![DACO Application](/assets/data-access/daco-application.png)
```

Comment:
Do you think it's better to move picture before the text? Probably I am lazy and find that looking at pic is more straightforward for me to understand the meaning. :-)

Reply:
good point haha

Reply:
I noticed that you have labelled the sections with numbers in the picture, if moving pic before the text, it will be easier for the user to make the connection.

Reply:
I put this image after #3 as the first two points are about logging in and the dashboard. Thanks LInda

Reply:
I agree with Linda, I was just thinking the same - it's easier to see the picture before the text.